__tests__/apiTests.test.js

```diff
@@ -164,4 +164,45 @@ describe("GET /api/articles/:article_id/comments",()=>{
                 expect(body.msg).toBe("Invalid query params!!")
             })
     })
+})
+
+describe("POST /api/articles/:article_id/comments",()=>{
+    test("get status code 200 and responds with the posted comment",()=>{
+        const comment = {username: "jonny",body: "This article is pretty great! :)"}
+        return request(app)
+
+            .post("/api/articles/2/comments")
+            .send(comment)
+            .expect(201)
+                .then(({body})=>{
+                    console.log(body)
+                    const postedComment = body.comment
+                    expect(postedComment).toMatchObject([{"article_id": 2, "author": "butter_bridge", "body": "This article is pretty great! :)", "comment_id": 19, "votes": 0}])
+                    expect(typeof postedComment[0].created_at).toBe("string")
```

Comment:
you could include created_at in the toMatchObject and instead do something like created_at: expect.any(String) or expect.toBeDateString()